### ISSUE TITLE
Remove extra margins (right) on small screens - React JS | Material UI

### ISSUE THREAD

**Issue by izhar360** [izhar360/BadBingFrontEnd#3]
Simple blog currently hosted on Netlify: https://determined-visvesvaraya-4f6a96.netlify.app/

please visit the blog wait for the articles to load, open the first one, check on small screen and see the extra right margins, i want to get rid of it

code => in the repo  /pages/posts/[id].js where the single Post code belongs.

feel free to ask anything!

if you want to work on it

download the code
npm install
npm run dev

**Comment by sambarnes90** [comment]
Do you want to assign me to this @izhar360 - I can work on this 😄

**Comment by izhar360** [comment]
> Do you want to assign me to this @izhar360 - I can work on this 😄 

Yes please go on, my code is not of very high standard so please ask anything...

**Comment by izhar360** [comment]
Only the first article have content "social media is not the problem we are" one

**Comment by sambarnes90** [comment]
Can you take a screenshot of what you are looking at @izhar360 ? 

The netlify link doesn't appear to have this behaviour?

**Comment by izhar360** [comment]
Its not fixed and moveable to right side, the current view is on samsung galaxy s8

**Comment by izhar360** [comment]
Its not fixed and moveable to right side, the current view is on samsung galaxy s8

![Screenshot_20211013-194753_Chrome.jpg](https://user-images.githubusercontent.com/79567009/137157828-05bb0be4-db63-4315-9197-190e7827ecb2.jpg)![Screenshot_20211013-194803_Chrome.jpg](https://user-images.githubusercontent.com/79567009/137157872-1b6ff69b-0c6a-40e6-b8c2-02881b343861.jpg)![Screenshot_20211013-194827_Chrome.jpg](https://user-images.githubusercontent.com/79567009/137157885-34f4f4b2-bcdf-428d-bc52-b6e48bf9c290.jpg)
